addon/sidebar.css

```diff
@@ -60,39 +60,53 @@ body {
   display: flex;
   flex-direction: column;
   margin-bottom: 16px;
+  width: 71vw;
+  max-width: 284px;
+  height: 52vw;
```

Comment:
Switch either width or height to "auto" so we don't have to think about aspect-ratio. I think one of these properties is being ignored right now anyway. And the same for below.